Cargo.toml

```diff
@@ -5,6 +5,7 @@ members = [
   "ingestion",
   "integration_tests",
   "rove_connector", "util",
+  "common",
```

Comment:
We already have the util crate we can reuse for this, rather than creating a new one

Reply:
I agree, will do! (I actually considered putting it under util first, but then I changed my mind for some reason)